sros2/test/sros2/commands/security/verbs/test_create_key.py

```diff
@@ -62,69 +81,106 @@ def verify_signature(cert, signatory):
     return True
 
 
-def check_cert_pem(path, signatory):
-    cert = load_cert(path)
+def test_create_key(node_keys_dir):
+    expected_files = (
+        'cert.pem', 'ecdsaparam', 'governance.p7s', 'identity_ca.cert.pem', 'key.pem',
+        'permissions.p7s', 'permissions.xml', 'permissions_ca.cert.pem', 'req.pem', 'request.cnf'
+    )
+    assert len(os.listdir(node_keys_dir)) == len(expected_files)
+
+    for expected_file in expected_files:
+        assert os.path.isfile(os.path.join(node_keys_dir, expected_file))
+
+
+def test_cert_pem(node_keys_dir):
+    cert = load_cert(os.path.join(node_keys_dir, 'cert.pem'))
     check_common_name(cert.subject, u'/test_node')
     check_common_name(cert.issuer, u'sros2testCA')
+
+    signatory = load_cert(os.path.join(node_keys_dir, 'identity_ca.cert.pem'))
     assert verify_signature(cert, signatory)
 
 
-def check_permissions_xml(path, signatory):
-    ElementTree.parse(path)
+def test_ecdsaparam(node_keys_dir):
+    # Would really like to verify the signature, but ffi just can't use
```

Comment:
Lucky for us we'll be removing this file entirely

Reply:
Whoops, that comment isn't actually valid for that particular test! I removed it. But yeah, looking forward to getting rid of that one.